oppia_android_test.bzl

```diff
@@ -1,8 +1,8 @@
 load("@rules_jvm_external//:defs.bzl", "artifact")
 load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
 
-def oppia_android_test(name, srcs, test_manifest, custom_package, resource_files,
-                       test_class, src_library_name):
+def oppia_android_test(name, srcs, test_manifest, custom_package, test_class, src_library_name,
+                       resource_files=None, assets=None, assets_dir=None):
```

Comment:
It will be helpful to add docs about these arguments also in the below `Args:` doc.

Reply:
Done!